src/components/ProjectTable.js

```diff
@@ -1,210 +1,69 @@
-                              className='btn btn-danger btn-fab btn-fab-mini btn-round'>
-                              <i className='material-icons'>delete</i>
-                            </button>
-                          </td>
-                        </tr>
-                        <tr>
-                          <td>6</td>
-                          <td>Mason Porter</td>
-                          <td>Chile</td>
-                          <td>Gloucester</td>
-                          <td className='text-warning'>$78,615</td>
-                          <td>
-                            <button
-                              type='button'
-                              className='btn btn-default btn-fab btn-fab-mini btn-round'>
-                              <i className='material-icons'>edit</i>
-                            </button>
-                          </td>
-                          <td>
-                            <button
-                              type='button'
-                              className='btn btn-danger btn-fab btn-fab-mini btn-round'>
-                              <i className='material-icons'>delete</i>
-                            </button>
-                          </td>
-                        </tr>
-                      </tbody>
-                    </table>
+export default class ProjectTable extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      projects: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://lesewert.herokuapp.com/api/v1/projects')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          projects: data.projects,
+        });
+      })
+      .catch(error => console.log(error));
+  }
+  render() {
+    return (
+      <div className='main-panel'>
+        <div className='content'>
+          <div className='container-fluid'>
+            <button type='button' className='btn btn-success'>
+              <i className='material-icons'>add_circle</i> New project
+            </button>
+            <div className='row'>
+              <div className='col-md-12'>
+                <div className='card'>
+                  <div className='card-header card-header-warning'>
+                    <h4 className='card-title '>Project Overview</h4>
+                    {/* <p className='card-category'>
+                      {' '}
+                      Here is a subtitle for this table
+                    </p> */}
+                  </div>
+                  <div className='card-body'>
+                    <div className='table-responsive'>
+                      <table className='table'>
+                        <thead className=' text-warning'>
+                          <th>ID</th>
+                          <th>Title</th>
+                          <th>Start</th>
+                          <th>End</th>
+                          <th>Participants</th>
+                          <th />
```

Comment:
And may be it's better to place buttons in the same column, because there will be a lot of space between them on big screen sizes. It also can help with disappeared remove button in small screen.

Reply:
I think it was better in separate columns, but I put it in one column now. I added {' '} this time on purpose to have some space between :)